kotlinpoet/src/main/java/com/squareup/kotlinpoet/LambdaTypeName.kt

```diff
@@ -50,10 +53,12 @@ public class LambdaTypeName private constructor(
     suspending: Boolean = this.isSuspending,
     tags: Map<KClass<*>, Any> = this.tags.toMap()
   ): LambdaTypeName {
-    return LambdaTypeName(receiver, parameters, returnType, nullable, suspending, annotations, tags)
+    return LambdaTypeName(receiver, contextReceivers, parameters, returnType, nullable, suspending, annotations, tags)
   }
 
   override fun emit(out: CodeWriter): CodeWriter {
+    out.emitContextReceivers(contextReceivers)
+    out.emit(" ")
```

Comment:
should this be with a space-preserving dot?

```suggestion
    out.emit("·")
```

Reply:
yes I missed that